### ISSUE TITLE
Parable views not working for developers with Windows systems

### ISSUE THREAD

**Issue by FaaPz** [devvoh/parable#1]
## Expected Behavior

A fresh installation of Parable on a Windows system should show the default view (`View/Home/index.phtml`).

## Current Behavior

The default Parable view (`View/Home/index.phtml`) shows the following output:
```html
<!DOCTYPE HTML>
<html>
    <head>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />

        <title><br />
<b>Fatal error</b>:  Uncaught ReflectionException: Class ParableC:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Auth\Authentication does not exist in C:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Framework\Toolkit.php:71
Stack trace:
#0 C:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Framework\Toolkit.php(71): ReflectionClass-&gt;__construct('ParableC:\\MAMP\\...')
#1 C:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Framework\Toolkit.php(90): Parable\Framework\Toolkit-&gt;loadResourceMap()
#2 C:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Framework\View.php(123): Parable\Framework\Toolkit-&gt;getResourceMapping('Config')
#3 C:\MAMP\htdocs\cgi_whereabouts_dashboard\app\View\Home\index.phtml(7): Parable\Framework\View-&gt;__get('config')
#4 C:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Framework\View.php(110): require('C:\\MAMP\\htdocs\\...')
#5 C:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Framework\View.p in <b>C:\MAMP\htdocs\cgi_whereabouts_dashboard\vendor\devvoh\parable\src\Framework\Toolkit.php</b> on line <b>71</b><br />
```

## Possible Solution

Our temporary fix in `Parable\Framework\Toolkit`:
```php
// $fullClassName = str_replace($this->path->getDir('vendor/devvoh/parable/src'), '', $file->getRealPath());
$fullClassName = str_replace($this->path->getDir('vendor\\devvoh\\parable\\src'), '', $file->getRealPath());
```

You might want to identify the development machine?

## Context

As a Mac user I had to code the whole project all by myself.
My team members (Windows users) couldn't do anything for a very very long time, because Windows.

Joke.

## Your Environment

* Version used: **Parable 0.8.14+**
* Environment name and version (e.g. Chrome 39, node.js 5.4): **Chrome 54**
* Operating System and version (desktop or mobile): **Windows 10 (x64)**
* Link to your project: -

**Comment by Mvannes** [comment]
Confirming. Have Windows machine, 
![image](https://cloud.githubusercontent.com/assets/8969371/20967894/ce97835e-bc81-11e6-84a8-b2aa641e9748.png)

Above is the error being thrown.

**Comment by devvoh** [comment]
Alright, that's something to fix. Will look into it tonight!

**Comment by devvoh** [comment]
0.8.18 fixes this - see 5b88477

I'm not currently able to verify the fix, but it should work. When the fix is verified I'll close this issue. If it doesn't work, first off - shame on me, second off, I'll have to set up a windows env to fix it on.

So let me know!

**Comment by devvoh** [comment]
If this still doesn't work, let me know. For now I'm closing this :)
